pkg/webui/lib/shared-messages.js

```diff
@@ -149,6 +149,9 @@ export default defineMessages({
   firmwareVersion: 'Firmware version',
   fNwkSIntKey: 'FNwkSIntKey',
   fNwkSIntKeyDescription: 'Forwarding network session integrity key',
+  frameCounterWidth: 'Frame counter width',
+  frameCounter16Bit: '16 bit',
+  frameCounter32Bit: '32 bit',
```

Comment:
```suggestion
  16Bit: '16 bit',
  32Bit: '32 bit',
```
And sort accordingly.